docs/run/native-gates.mdx

```diff
@@ -0,0 +1,104 @@
+| [measure](/api/qiskit/qiskit.circuit.library.Measure) | |
+| [delay](/api/qiskit/qiskit.circuit.Delay) |  |
+
+### Eagle
+
+| Name |  Notes |
+| :--- | :--- |
+| [ECR](/api/qiskit/qiskit.circuit.library.ECRGate) | two-qubit gate |
+| [RZ](/api/qiskit/qiskit.circuit.library.RZGate) | single-qubit gate |
+| [SX](/api/qiskit/qiskit.circuit.library.SXGate)  |single-qubit gate |
+| [X](/api/qiskit/qiskit.circuit.library.XGate)  | single-qubit gate |
+| [ID](/api/qiskit/qiskit.circuit.library.IGate) | single-qubit gate wait cycle |
+| [reset](/api/qiskit/qiskit.circuit.library.Reset)  | single-qubit gate, non-unitary; not the same as the initialization done at the start of a circuit to prepare the all 0's state |
+| [if_else](/api/qiskit/qiskit.circuit.IfElseOp) | control flow for classical feedforward |
+| [for_loop](/api/qiskit/qiskit.circuit.ForLoopOp) | control flow for classical feedforward |
+| [switch_case](/api/qiskit/qiskit.circuit.SwitchCaseOp) | control flow for classical feedforward |
+| [measure](/api/qiskit/qiskit.circuit.library.Measure) | |
+| [delay](/api/qiskit/qiskit.circuit.Delay) |  |
+
+### Falcon
+
+| Name |  Notes |
+| :--- | :--- |
+| [CX](/api/qiskit/qiskit.circuit.library.CXGate)  | two-qubit gate |
+| [RZ](/api/qiskit/qiskit.circuit.library.RZGate) | single-qubit gate |
+| [ID](/api/qiskit/qiskit.circuit.library.IGate) | single-qubit gate wait cycle |
+| [reset](/api/qiskit/qiskit.circuit.library.Reset)  | single-qubit gate, non-unitary; not the same as the initialization done at the start of a circuit to prepare the all 0's state |
+| [if_else](/api/qiskit/qiskit.circuit.IfElseOp) | control flow for classical feedforward |
+| [for_loop](/api/qiskit/qiskit.circuit.ForLoopOp) | control flow for classical feedforward |
+| [switch_case](/api/qiskit/qiskit.circuit.SwitchCaseOp) | control flow for classical feedforward |
+| [measure](/api/qiskit/qiskit.circuit.library.Measure) | |
+| [delay](/api/qiskit/qiskit.circuit.Delay) |  |
+
+
+<Admonition type="tip" title="The init_qubits flag">
+
+The `init_qubits` flag, set as a [primitive execution option,](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.ExecutionOptions) controls whether qubits are reset to the zero state at the start of each circuit. Its default value is True, indicating that the qubits should be reset. If False, the qubits will begin in the final state from the previous shot, and you must manually insert [resets](/api/qiskit/qiskit.circuit.library.Reset) if you want to reset them to the zero state. If a job consists of multiple circuits, then the shots are executed in a "round-robin" fashion. That is, each circuit will be executed in sequence to obtain one shot from each circuit. This process is then repeated until the requested number of shots has been obtained from all circuits.
+</Admonition>
+
+## Next steps
+
+<Admonition type="tip" title="Recommendations">
```

Comment:
one of our goals is to have every docs page include a link to a learning platform topic where users can start using these concepts in practice, can you add a link to a sensible learning platform course/tutorial here please?